kin-sdk-core/src/androidTest/java/kin/sdk/core/KinAccountTest.java

```diff
@@ -179,17 +163,10 @@ public void sendTransactionSync_SecondTimeEmptyPassphraseFails() throws Exceptio
         Balance afterBalance = senderAccount.getBalanceSync();
         assertTrue((senderBalance.value().subtract(amountToSend).compareTo(afterBalance.value())) == 0);
 
-        Exception exception = null;
-        try {
-            senderAccount.sendTransactionSync(kinAccount.getPublicAddress(), "", new BigDecimal(1));
-        } catch (Exception e) {
-            exception = e;
-        }
-        assertNotNull(exception);
-        assertTrue(exception instanceof PassphraseException);
+        senderAccount.sendTransactionSync(kinAccount.getPublicAddress(), "", new BigDecimal(1));
     }
 
-    @Test
+    @Test(expected = PassphraseException.class)
```

Comment:
you changed the test this way.
The reason it's on the second sendTransaction because the only second transaction should fail.

As the code now, can fail on the first sendTransaction and its still will pass.

Reply:
oops, you're right

Reply:
👍

Reply:
fixed